CPP_STYLE_GUIDE.md

```diff
@@ -38,19 +38,24 @@ runtime features.
 Coding guidelines are based on the following guides (highest priority first):
 1. This document
 2. The [Google C++ Style Guide][]
-3. The ISO [C++ Core Guidelines][]
 
-In general code should follow the C++ Core Guidelines, unless overridden by the
-Google C++ Style Guide or this document. At the moment these guidelines are
-checked manually by reviewers, with the goal to validate this with automatic
-tools.
+In general code should the Google C++ Style Guide, unless overridden by this
```

Comment:
typo? `should the Google`?

Reply:
(I guess `follow` is missing in the sentence?)

Reply:
I'd drop `In general` if possible. It serves only to make following the style guide seem optional. Even if it is optional, `should` seems to provide sufficient wiggle-room for those times when you can't follow it.